metadata/demographic/urban_areas_census_2020.md

```diff
@@ -0,0 +1,72 @@
+# Title
+
+Utah Urban Areas Census 2020
+
+## ID
+
+8d75fea5-645a-44ae-b066-1d231dab3399
+
+## Brief Summary
+
+Polygon dataset of areas designated as residential, commercial, or other urban land uses.
+
+## Summary
+
+This dataset contains polygons representing urban areas as defined by the [US Census Bureau](https://www.census.gov/) during the 2020 Census. This dataset contains boundary information only and does not include demographic data.
```

Comment:
```suggestion
This dataset contains polygons representing urban areas as defined by the [US Census Bureau](https://www.census.gov/) during the 2020 Census. This dataset contains boundary information only and is intended to be joined to other demographic data tables.
```

If the original language matches the other census boundary data, lets keep that. If not, It may be helpful to reference why it's just boundary information.